internal/cluster/cluster.go

```diff
@@ -177,6 +178,18 @@ func (m *Manager) RegisterAddHostsOCPCluster(c *common.Cluster, db *gorm.DB) err
 }
 
 func (m *Manager) DeregisterCluster(ctx context.Context, c *common.Cluster) error {
+
+	var metricsErr error
+	for _, h := range c.Hosts {
+		if err := m.hostAPI.ReportValidationFailedMetrics(ctx, h, c.OpenshiftVersion, c.EmailDomain); err != nil {
+			m.log.WithError(err).Errorf("Failed to report metrics for failed validations on host %s in cluster %s", h.ID, c.ID)
+			metricsErr = multierror.Append(metricsErr, err)
+		}
+	}
+	if metricsErr != nil {
```

Comment:
I'm not sure that we want to fail the DeregisterCluster operation because we failed to send metrics - @avishayt WDYT?

Reply:
Duplication of https://github.com/openshift/assisted-service/pull/885#discussion_r559199985.
Closing.